dialplan/asterisk/extensions_lib_user.conf

```diff
@@ -239,7 +239,9 @@ same  =      n,Hangup()
 
 [usersharedlines]
 ; exten should be a user UUID
-exten = _[0-9a-f].,1,AGI(agi://${XIVO_AGID_IP}/get_user_interfaces,${EXTEN})
+exten = _[0-9a-f].,1,NoOp(Ringing lines of user ${EXTEN})
+same  = n,CELGenUserEvent(WAZO_ORIGINATE_ALL_LINES,user_uuid:${EXTEN},tenant_uuid:${WAZO_TENANT_UUID})
```

Comment:
I don't know xivo-config... the `EXTEN` is the user uuid ?

Reply:
Yes, as indicated by the comment at line 241. When calld originates a call(from an API request) ringing all lines of a user for the user to choose which line to use, the originated call is forwarded to this dialplan context, with the extension being the user uuid(`{user_uuid}@usersharedlines`).